app/Console/Commands/RemoveExplicitEols.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace App\Console\Commands;
+
+use App\Models\Asset;
+use App\Models\AssetModel;
+use Illuminate\Console\Command;
+
+class RemoveExplicitEols extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'snipeit:remove-explicit-eols {--model_name=}';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Removes explicit EOLs on assets with selected model so they may inherit the asset model EOL';
+
+    /**
+     * Execute the console command.
+     */
+    public function handle()
+    {
+        $assetModel= AssetModel::where('name', '=', $this->option('model_name'))->first();
```

Comment:
How is this going to handle spaces in the model name? Should we perhaps pass it an ID instead? Maybe optionally, so we can reset all if they have a lot of models and want to handle it for all asset models.

Reply:
The way you enter a model name is `--model_name="Spaces in Between the quotes are fine"`
I though about doing finding the model by Id, but if you put in the wrong id number you could do a lot of damage. if you misspell the model name it returns "Asset Model not Found".

Reply:
i could write an all option